build.gradle

```diff
@@ -114,11 +116,11 @@ pitest {
     mutationThreshold = 75
 }
 
-sonarqube {
+sonar {
```

Comment:
is there a reason for the task name change?

Reply:
sonarqube came up as a deprecated task in gradle